### ISSUE TITLE
How can i put more databases in river.toml?

### ISSUE THREAD

**Issue by caiobiodere** [siddontang/go-mysql-elasticsearch#262]
I have been working on these references:
```
[[source]]
schema = "test"
tables = ["testing"]
```

But when I put something new like this:
```
[[source]]
schema = "test"
tables = ["testing"]
[[source]]
schema = "test2"
tables = ["testing"]
```

That's not working anymore because the plugin can't sync my new database (test2) into ElasticSearch, just thinking about binlog stuffs because it should be a problem with logs.

<img width="1416" alt="screen shot 2018-06-14 at 12 06 35 pm" src="https://user-images.githubusercontent.com/1935649/41420698-746eb718-6fcb-11e8-9a84-7434bc42c953.png">

**Comment by paladini** [comment]
Same issue here.

**Comment by siddontang** [comment]
hi @caiobiodere 

Now it can't support, but seem it is not hard, if I have time, I will consider it.

**Comment by caiobiodere** [comment]
The real problem is when I put a new database in river.toml.

I started my first time using one datatable but when I put a new one and tried again that's not worked.

**Comment by siddontang** [comment]
does the new database has already been created before your first time running? if yes, your old data can't be synced even after you add the database in river.toml, only the new changed data can be synced. 

If the new changed data still can't be synced, there may be something wrong.

**Comment by caiobiodere** [comment]
The new database isn't already created before my first time running, that's the problem because when I put something new the plugin can't understand my new database and put there.

If the new changed data still can't be synced, there may be something wrong. Exactly the problem.

**Comment by caiobiodere** [comment]
@siddontang you could use this test case ->

1 - Create river.toml using one database and sync for the first time.
2 - Close that and change river.toml adding a new database and sync again, you will see the problem because the plugin isn't identifying a new database and putting these changes into elastic.

**Comment by siddontang** [comment]
@caiobiodere 
I try to reproduce your problem but fail with your step. Seem that the newly changed data in the new database can still be changed. 

Btw, you use the same table name testing in the source and assign no special rules for them, so both these databases' testing tables will be synced to ES as `/testing/testing`,  with the same doc and type.

**Comment by caiobiodere** [comment]
@siddontang Yes I know what are you talking about, btw I need to use for each index a new database with tables.

Do you know how can I do that?

**Comment by siddontang** [comment]
> btw I need to use for each index a new database with tables

can you explain more detailedly or give a pseudo example?
